general-concepts/licenses/text.xml

```diff
@@ -6,8 +6,28 @@
 <body>
 <p>
 All ebuilds must specify a <c>LICENSE</c> (note the American English
-spelling) variable. The value of this variable must be the name of a
-file in the Gentoo repository's <c>licenses/</c> directory.
+spelling) variable. The license names listed in this variable must
+match files existing in the repository's <c>licenses/</c> directory.
+</p>
+
+<p>
+The value of this variable should include all licenses pertaining
+to the files installed by the package. If some parts of the package
+are installed only conditionally, or their license depends on the USE
+flag combination, you can use USE conditionals in <c>LICENSE</c>:
+</p>
+
+<codesample lang="ebuild">
+LICENSE="LGPL-2.1+ tools? ( GPL-2+ )"
+</codesample>
+
+<p>
+If the package sources include additional files that are not installed,
+their license should not be listed. However, if those files are used
+at build time, then the license must not impose any restrictions that
+could prevent some of the Gentoo users from building the software.
```

Comment:
```suggestion
could prevent users from building the software.
```

"some of the Gentoo users" is vague

Reply:
The idea was to emphasize that it might impact only some of the users (e.g. have business use restrictions, country restrictions etc.). But I guess either way works.